agent/custom/bbc_action.py

```
@@ -635,8 +643,61 @@ def _wait_for_battle_end(self, state: dict):
+                        if stop_result.get('success'):
+                            mfaalog.info("[ExecuteBbcTask] BBC脚本已停止")
+                        else:
+                            mfaalog.warning(f"[ExecuteBbcTask] 停止BBC脚本失败: {stop_result.get('error', '')}")
+                        # 等待BBC脚本完全停止，避免干扰后续任务
+                        time.sleep(3)
+                        # 正常结束不设置弹窗信息，避免被误判为异常
+                        state['finished'] = True
+                        break
+                else:
+                    miss_count = 0
+                    mfaalog.info(f"[ExecuteBbcTask] 未识别到错误提示，战斗进行中，识别文本: {all_texts}")
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)